cpp/src/barretenberg/honk/sumcheck/sumcheck_round.test.cpp

```diff
@@ -36,7 +36,7 @@ static Univariate<FF, max_relation_length> compute_round_univariate(
     const RelationParameters<FF>& relation_parameters,
     const FF alpha)
 {
-    size_t round_size = 1;
+    size_t round_size = 2;
```

Comment:
This was always a bug but it didnt matter because it was being used in loop like this `for (size_t edge_idx = 0; edge_idx < round_size; edge_idx += 2)` in `compute_univariate` Round size is at minimum 2, never 1 in practice.

Reply:
This may have been left over from when we thought of round size as the number of edges. Its not that anymmore. Its the number of rows ( = number of edges * 2)